### ISSUE TITLE
Add FCREPO_DEVELOPMENT_CONTEXT environmental variable and SOLR_DEVELOPMENT_CORE

### ISSUE THREAD

**Issue by ksclarke** [UCLALibrary/packer-samvera#27]
The VM uses the Ansible roles to provision so the `fedora.yml` what's in the `californica` repo doesn't quite match up. We need to parameterize the fedora.yml to take a customizable Tomcat context, while still using `/rest` as a default.

What's there:
```
vagrant@localhost:~/californica$ cat config/fedora.yml
development:
  user: fedoraAdmin
  password: fedoraAdmin
  url: http://127.0.0.1:<%= ENV['FCREPO_DEVELOPMENT_PORT'] || 8080 %>/rest
  base_path: /dev
test:
  user: fedoraAdmin
  password: fedoraAdmin
  url: http://127.0.0.1:<%= ENV['FCREPO_TEST_PORT'] || 8986 %>/rest
  base_path: /test
production:
  user: fedoraAdmin
  password: fedoraAdmin
  url: http://127.0.0.1:8080/fedora/rest
  base_path: /prod
vagrant@localhost:~/californica$ lynx http://127.0.0.1:8080/fedora/rest
```
We need something like: `url: http://127.0.0.1:<%= ENV['FCREPO_DEVELOPMENT_PORT'] || 8080 %><%= ENV['FCREPO_DEVELOPMENT_CONTEXT'] || /rest` (there is a separate ticket in californica for this: uclalibrary/californica#59).

Once this is done we just need to add the FCREPO_DEVELOPMENT_CONTEXT parameter to this Packer build.

Related #28

**Comment by ksclarke** [comment]
The main question I think is are there other Tomcat contexts other than Fedora in the Ansible build. If there are, we definitely need to parameterize the Californica config instead of the roles.

**Comment by ksclarke** [comment]
I think it's just best to update 'californica' to parameterize FCREPO_DEVELOPMENT_CONTEXT, with a default of '/rest'. Packer can override that to be 'fedora/rest'

**Comment by ksclarke** [comment]
The solr.yml also needs to be parameterized so Packer will need to supply the value for that parameter once that's done.

**Comment by ksclarke** [comment]
Bess made a good suggestion to just use a single parameter for port and context/core. Do that instead.
